lib/project-data-updater.js

```diff
@@ -22,8 +22,7 @@ module.exports.isValidUpdate = function(info) {
 module.exports.fileUpdated = function(commits) {
   for (var i = 0; i != commits.length; i++) {
     var commit = commits[i];
-    if ((commit.added.indexOf(module.exports.ABOUT_YML) !== -1) ||
-      (commit.modified.indexOf(module.exports.ABOUT_YML) !== -1)) {
+    if (commit.added.length + commit.modified.length > 0) {
```

Comment:
Actually, I think the original behavior is correct: We only want to process webhooks where `.about.yml` specifically is either added or modified, not every webhook that contains new files or modifications. The only thing we need to change is emitting the error message when this returns `false`, because the server will receive webhooks for every 18F change, not just `.about.yml` changes.